### DISCUSSION TITLE
Is there a way to organise files in different notebooks like in OneNote?

### DISCUSSION THREAD

**Discussion by tinywretch** [xournalpp/xournalpp#4856] [Q&A]
I've been looking for a solid alternative to bamboo paper or onenote for linux because I'm tired of windows, however, I use my surface pro 8 to take notes with the surface pen and while Xournal is a great app for note taking, I haven't been able to figure out if you can set up multiple notebooks for different subjects and access them whenever I want.

**Comment by netlimpopo** [comment]
Hello! 
Simple answer: **no**. One can not organise the files into notebooks from within the Xournal++ app yet. However one can set up folders in the native file manager (Win: explorer, Mac: finder,...) to organise notes. This file manager usally allows several additional feature such as tags, last changes, importance, etc. . There are discussions about implementing notebooks in Xournal++ though. Atm there is no open pull request for it. 
At some point it was considered to integrate files within Joplin to build up a hierachical tree and reference files within notebooks, which is still annoying as Joplin can't read those files. 
Nevertheless Notebooks in my opinion only make sense for 3 reasons: 
1) for a global seach all across files to find the one aspect you are looking for. 
2) if one can link between files of notebooks (for reference reasons)
3) To tag content inside one file, then again with the power of a global searchbar, find tags of the same name.

If the above points are not applicable, then there is no real benefit of notebooks over folders with files. 
(Prove me wrong if I am missing something). 
I usally get frustrated when I try to extract Information out of OneNote, es specially pages of notebooks, as the simply are not just files within folders, but maybe that's just me trying to extract Information from a vendor-lock in product. 🤷‍♂️. 

If you do don't need to use the keyboard for typing and want a handwriting only app there is also rnote oder stylus labs write.
